releases/release-5.0.1.md

```diff
@@ -0,0 +1,86 @@
+    - Users now need both INSERT and DELETE privileges on a table to perform REPLACE [#23939](https://github.com/pingcap/tidb/pull/23939)
+    - Fix performance regression of point select [#24070](https://github.com/pingcap/tidb/pull/24070)
+
++ TiKV
+
+    - Fix IN expr(coprocessor) didn't handle unsigned/signed int properly [#10018](https://github.com/tikv/tikv/pull/10018)
+    - Fix the issue that there are lots of empty regions after batch ingest. [#10015](https://github.com/tikv/tikv/pull/10015)
+    - Fix potential panics when input of cast_string_as_time is invalid UTF-8 bytes [#9995](https://github.com/tikv/tikv/pull/9995)
+    - Fix the bug that TiKV cannot startup when the end of file dict file is damaged. [#9992](https://github.com/tikv/tikv/pull/9992)
+
++ TiFlash
+
+    - Fix a potential issue that the DeleteRange in the storage engine failed to remove some data. [#1790](https://github.com/pingcap/tics/pull/1790)
+    - Fix problem that TiFlash coprocessor's cast time as int function may produce incorrect result. [#1786](https://github.com/pingcap/tics/pull/1786)
+    - Fix the bug that receiver cannot find tasks within 10s. [#1784](https://github.com/pingcap/tics/pull/1784)
+    - Fix potential invalid iterator in cancelMPPQuery [#1777](https://github.com/pingcap/tics/pull/1777)
+    - Fix problem that behavior of TiFlash coprocessor's bitwise operator is different from TiDB [#1774](https://github.com/pingcap/tics/pull/1774)
+    - Fix false alert of overlapped ranges if prefix next key is used [#1771](https://github.com/pingcap/tics/pull/1771)
+    - Fix the problem that TiFlash coprocessor's cast string as int function may produce incorrect result. [#1768](https://github.com/pingcap/tics/pull/1768)
+    - Fix the issue that continuous and fast writing may make TiFlash OOM [#1739](https://github.com/pingcap/tics/pull/1739)
+    - Fix bug that duplicated column name will make TiFlash throw error. [#1732](https://github.com/pingcap/tics/pull/1732)
+    - Support complex query plan tree with a max depth of 100. [#1722](https://github.com/pingcap/tics/pull/1722)
+    - Fix potential NPE in schema sync service when database is dropped between GC and getting the database info. [#1708](https://github.com/pingcap/tics/pull/1708)
+    - Fix the crash that causes by applying Raft commands to dropped tables [#1702](https://github.com/pingcap/tics/pull/1702)
+    - Fix the issue that TiFlash may panic during br restore [#1698](https://github.com/pingcap/tics/pull/1698)
+
+## 下面是未分类的 notes。请对以下 notes 分为 [New Features](#new-features)、[Compatibility Changes](#compatibility-changes)、[Improvements](#improvements)、[Bug Fixes](#bug-fixes) 四类，并移动到上面对应的标题下面。如果 Notes 多余，请进行移除。
+
++ PD
+
+    - Modify score calculator to satisfy isomerous stores [#3605](https://github.com/pingcap/pd/pull/3605)
+    - Avoid unexpected statistic modify after adding scatter region scheduler [#3602](https://github.com/pingcap/pd/pull/3602)
+
++ Tools
+
```

Comment:
- Lightning

* Fix the bug that the table count in the progress log is wrong. [#1005](https://github.com/pingcap/br/pull/1005)